src/locale/locales/de/messages.po

```diff
@@ -599,19 +565,19 @@ msgstr "Meine E-Mail ändern"
 
 #: src/view/screens/Settings/index.tsx:732
 msgid "Change password"
-msgstr ""
+msgstr "Passwort ändern"
 
 #: src/view/screens/Settings/index.tsx:741
 msgid "Change Password"
-msgstr ""
+msgstr "Passwort ändern"
 
 #: src/view/com/composer/select-language/SuggestedLanguage.tsx:73
 msgid "Change post language to {0}"
-msgstr ""
+msgstr "Beitragssprache in {0} ändern"
```

Comment:
Is "post" translated with "Beitrag" anywhere else? Or left as "Post"? I'm fine with both.

Reply:
"Beitrag" seems to have been what the previous translators decided worked best for "post", so I thought it best to stick with that.